### DISCUSSION TITLE
using custom credentials

### DISCUSSION THREAD

**Discussion by gohil-satyarthsinh** [zinclabs/zinc#560] [Q&A]
HI,

I recently did a POC for zinc search and used the credentials on the official website during the testing. Now that I am good to use it, I tried using my custom credentials for ZINC_FIRST_ADMIN_USER and ZINC_FIRST_ADMIN_PASSWORD. I used them in secrets and referred them in the YAML. But I am now able to login using the new credentials with the newly deployed setup. Please let me know if I am doing something wrong. Basically I need one user to admin, one to push logs and other users to view logs. Let me know if there is a way using which I should proceed.

**Comment by jonas-grgt** [comment]
ZINC_FIRST_ADMIN_USER and ZINC_FIRST_ADMIN_PASSWORD should be set as environment variables on the shell running the zinc process.

If anything fails creating the initial admin user it should appear in the logs. Look for: `init first user`
